api/v2/Routes/students/projections/index.js

```diff
@@ -23,12 +27,10 @@ router.get('/', async (req, res) => {
     const maxPointsSoFar = getMaxPointsSoFar(userGrades, maxScores);
 
     return res.status(200).json({
-        "zeros": Math.round(studentTotalScore),
-        "pace": Math.round((studentTotalScore / maxPointsSoFar) * maxPoints),
-        "perfect": Math.round(studentTotalScore + (maxPoints - maxPointsSoFar))
+        zeros: Math.round(studentTotalScore),
+        pace: Math.round((studentTotalScore / maxPointsSoFar) * maxPoints),
+        perfect: Math.round(studentTotalScore + (maxPoints - maxPointsSoFar)),
```

Comment:
nit: do we need a comma if this is the last entry? Or is the comma optional? What's the standard convention for this?

Reply:
This is just a standard that I personally like so I included it in my lint config since I'm the primary maintainer.